test/credentials_test.dart

```diff
@@ -36,4 +36,61 @@ Future<void> main([List<String>? args]) async {
     final credentials = Credentials.emailPassword("test@email.com", "000000");
     expect(credentials.provider, AuthProviderType.emailPassword);
   });
+
+  baasTest('Email/Password - register user confirmation throws', (configuration) async {
+    final app = App(configuration);
+    final authProvider = EmailPasswordAuthProvider(app);
+    String emailPrefix = generateRandomString(5);
+    String username = "$emailPrefix@bar.com"; // Usernames that don't contains 'realm_tests_do_autoverify' are not confirmed
+    String password = "SWV23R#@T#VFQDV";
+    expect(() async {
+      await authProvider.registerUser(username, password);
+    }, throws<RealmException>("failed to confirm user"));
+  });
+
+  baasTest('Email/Password - register user', (configuration) async {
+    final app = App(configuration);
+    final authProvider = EmailPasswordAuthProvider(app);
+    String emailPrefix = generateRandomString(5);
+    String username = "realm_tests_do_autoverify$emailPrefix@bar.com";
+    String password = "SWV23R#@T#VFQDV";
+    await authProvider.registerUser(username, password);
+    final user = await app.logIn(Credentials.emailPassword(username, password));
+    await app.logout(user);
+    await app.removeUser(user);
```

Comment:
By some reason the user is not removed from the app. Should be investigated.

Reply:
We probably don't these here and we'll write separate tests for user.logout/app.removeUser.

Reply:
We probably don't these here and we'll write separate tests for user.logout/app.removeUser.

Reply:
Yes sure.